media-gfx/qimgv/qimgv-0.9.1.ebuild

```diff
@@ -24,6 +24,7 @@ DEPEND="
 	dev-qt/qtwidgets:5
 	exif? ( media-gfx/exiv2:= )
 	kde? ( kde-frameworks/kwindowsystem:5 )
+	opencv? ( media-libs/opencv:0 )
```

Comment:
Please check linking to opencv, you most likely need to trigger rebuild on opencv subslot bump.

Reply:
> Please check linking to opencv, you most likely need to trigger rebuild on opencv subslot bump.
What did you mean by check? I'm not very familiar with `opencv`.
I changed it to `:=`, so it should rebuild now right?

Reply:
This is not unique to opencv. For dependencies that declare a subslot, we must check its meaning (often in a comment beside the `SLOT` declaration, or in `metadata.xml` even, then check if our package meets the criteria (most of the time it is about linking to a versioned soname).